### DISCUSSION TITLE
Smoothing

### DISCUSSION THREAD

**Discussion by Lenergyy** [pyvista/pyvista#2010] [Q&A]
Is there a way to load a custom .stl file, smooth the edges and surfaces (for example with surf.smooth(n_iter=100)) and save it as .stl?

Something like this:

![image](https://user-images.githubusercontent.com/97316670/148657774-43edff52-e79f-4c81-8824-3a7b39407e2c.png)

=>

![image](https://user-images.githubusercontent.com/97316670/148657798-5db6b269-c6b8-485c-be00-742199f5ab46.png)

**Comment by banesullivan** [comment]
```py
import pyvista as pv

mesh = pv.read('mymesh.stl')
```

should load your file. Then follow the smoothing example with that `mesh` object
